bika/lims/content/analysisrequest.py

```diff
@@ -1852,6 +1851,8 @@ def _getAnalysesNum(self):
                 total += 1
         return verified,total
 
+    security.declareProtected(View, 'getResponsible')
+
```

Comment:
Not important, but better use ``@security.public`` decorator instead of ``security.declareProtected``. Clener and easier to remember.
Example: https://github.com/naralabs/bika.lims/blob/wip/bika/lims/content/analysis.py#L27